src/scss/_admonitions.scss

```diff
@@ -1,8 +1,8 @@
 // Admonitions CSS originally inspired by https://squidfunk.github.io/mkdocs-material/getting-started/
 
-.admonition {
+div.admonition, .admonition{
```

Comment:
I added a more specific selector `div.admonition`, because this is how admonition styling is specified in `basic.css`. I leave the more catch-all selector for compatibility and in case *someone* (me) wants to reuse the styling for colapsible admonitions that use a `<details>` tag.